src/response_parser.ts

```diff
@@ -22,4 +24,40 @@ export default class ResponseParser {
 
     return res
   }
+
+  transformAnnotationResponse(options, data) {
+    const rows = data.data;
+    const columns = data.meta;
+    const result = [];
+    let hasTime = false;
+
+    for (let i = 0, len = columns.length; i < len; i++) {
+      const column = columns[i];
+
+      if (column.name === 'time') {
+        hasTime = true;
+        break;
+      }
+    }
+
+    if (!hasTime) {
+      return this.$q.reject({
+        message: 'Missing mandatory time column in annotation query.',
+      });
+    }
+
+    for (let i = 0, len = rows.length; i < len; i++) {
+      const row = rows[i];
+
+      result.push({
+        annotation: options.annotation,
+        time: Math.floor(row.time),
+        title: row.title,
+        text: row.text,
+        tags: row.tags ? row.tags.trim().split(/\s*, \s*/) : []
```

Comment:
There is a space in expression for splitting tags while `Show help` tooltip says `This is a comma separated string of tags e.g. 'tag1,tag2'`. It's misleading